syntax = "proto3";

message ClientRequest{
    int32 cmdType = 1;
    string clientId = 2;
    string authCode = 3;
    string serverId = 4;
    string r1 = 5;
}
